interactions/__init__.py

```diff
@@ -1,7 +1,360 @@
-from .client import *
-from .client.const import *
-from .models import *
-from .api import *
+from .client import (
+    __version__,
+    __repo_url__,
+    __py_version__,
+    __api_version__,
+    get_logger,
+    logger_name,
+    kwarg_spam,
+    DISCORD_EPOCH,
+    ACTION_ROW_MAX_ITEMS,
+    SELECTS_MAX_OPTIONS,
+    SELECT_MAX_NAME_LENGTH,
+    CONTEXT_MENU_NAME_LENGTH,
+    SLASH_CMD_NAME_LENGTH,
+    SLASH_CMD_MAX_DESC_LENGTH,
+    SLASH_CMD_MAX_OPTIONS,
+    SLASH_OPTION_NAME_LENGTH,
+    EMBED_MAX_NAME_LENGTH,
+    EMBED_MAX_DESC_LENGTH,
+    EMBED_MAX_FIELDS,
+    EMBED_TOTAL_MAX,
+    EMBED_FIELD_VALUE_LENGTH,
+    Singleton,
+    Sentinel,
+    GlobalScope,
+    Missing,
+    MentionPrefix,
+    GLOBAL_SCOPE,
+    MISSING,
+    MENTION_PREFIX,
+    PREMIUM_GUILD_LIMITS,
+    Absent,
+    T,
+    T_co,
+    Client,
+    AutoShardedClient,
+    smart_cache,
+    errors,
+    utils,
+)
+from .client.const import (
+    __version__,
+    __repo_url__,
+    __py_version__,
+    __api_version__,
+    get_logger,
+    logger_name,
+    kwarg_spam,
+    DISCORD_EPOCH,
+    ACTION_ROW_MAX_ITEMS,
+    SELECTS_MAX_OPTIONS,
+    SELECT_MAX_NAME_LENGTH,
+    CONTEXT_MENU_NAME_LENGTH,
+    SLASH_CMD_NAME_LENGTH,
+    SLASH_CMD_MAX_DESC_LENGTH,
+    SLASH_CMD_MAX_OPTIONS,
+    SLASH_OPTION_NAME_LENGTH,
+    EMBED_MAX_NAME_LENGTH,
+    EMBED_MAX_DESC_LENGTH,
+    EMBED_MAX_FIELDS,
+    EMBED_TOTAL_MAX,
+    EMBED_FIELD_VALUE_LENGTH,
+    Singleton,
+    Sentinel,
+    GlobalScope,
+    Missing,
+    MentionPrefix,
+    GLOBAL_SCOPE,
+    MISSING,
+    MENTION_PREFIX,
+    PREMIUM_GUILD_LIMITS,
+    Absent,
+    T,
+    T_co,
+)
+from .models import (
+    misc,
```

Comment:
I'm not sure why we're importing other namespaces when we expose everything from them underneath

Reply:
That looks like an error with inferencing - I'll see if I can filter that out programmatically